core/templates/dev/head/pages/admin/admin.html

```diff
@@ -105,7 +106,6 @@
     <script src="{{TEMPLATE_DIR_PREFIX}}/services/explorationContextService.js"></script>
     <script src="{{TEMPLATE_DIR_PREFIX}}/services/utilsService.js"></script>
 
-    <script src="{{TEMPLATE_DIR_PREFIX}}/components/forms/FormBuilder.js"></script>
```

Comment:
Hi @MAKOSCAFEE -- was removing this intentional? The admin page does not load when I check out this branch.